src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1965,6 +1971,35 @@ public void cannotCreateRequestForClaimedReturnedItem(String requestType) {
     )));
   }
 
+  @Test
+  public void canFetchHundredRequests() {
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    final List<IndividualResource> createdRequests =
+      IntStream.range(0, 100).mapToObj(notUsed -> requestsFixture.place(
+        new RequestBuilder()
+          .open()
+          .page()
+          .forItem(itemsFixture.basedUponSmallAngryPlanet())
+          .by(usersFixture.charlotte())
+          .fulfilToHoldShelf()
+          .withPickupServicePointId(pickupServicePointId)))
```

Comment:
I might extract the placing of the request into a method, to reduce the noise in this test.